src/liquid/components/ld-toggle/ld-toggle.css

```diff
@@ -0,0 +1,83 @@
+.ld-toggle {
+  background-color: var(--ld-col-rblck4);
+  border-radius: 1rem;
+  border: 0;
+  display: inline-flex;
+  height: 2rem;
+  padding: 0;
+  position: relative;
+  width: 3.375rem;
+
+  &:where(:disabled, [aria-disabled='true']) {
+    background-color: var(--ld-col-bg-g);
+  }
+
+  &:where(:not(:disabled):not([aria-disabled='true'])) {
+    cursor: pointer;
+  }
+
+  .ld-toggle__knob {
+    background-color: var(--ld-col-wht);
+    border-radius: 50%;
+    display: inline-block;
+    height: 1.75rem;
+    margin: 2px;
+    pointer-events: none;
+    transition: transform 200ms ease-in-out;
+    width: 1.75rem;
+  }
+
+  &:where([aria-checked='true']) {
+    .ld-toggle__knob {
+      transform: translateX(calc(1.375rem));
```

Comment:
Here you could use either a relative unit or a CSS custom prop with `calc`. Something like `translateX(calc(0.5 * var(--ld-toggle-width)))` or simply using some percentage value.